build/helper/documentation_snippets.py

```diff
@@ -6,6 +6,17 @@
 You can specify a subset of repeated capabilities using the Python index notation on an
 {0}.Session instance, and calling this method on the result.:
 '''
+
+rep_cap_attr_desc = '''
+This property can use repeated capabilities (usually channels). If set or get directly on the
+{0}.Session object, then the set/get will use all repeated capabilities in the session.
+You can specify a subset of repeated capabilities using the Python index notation on an
+{0}.Session instance, and calling set/get value on the result.:
+
+    session['0,1'].{0} = var
```

Comment:
This is not how things are handled anymore in master.

Reply:
Fixed